BuildPropertiesPlugin/plugin/src/main/groovy/com/novoda/buildproperties/BuildPropertiesPlugin.groovy

```diff
@@ -9,8 +9,10 @@ class BuildPropertiesPlugin implements Plugin<Project> {
     @Override
     void apply(Project project) {
         boolean isAndroidApp = project.plugins.hasPlugin('com.android.application')
-        if (!isAndroidApp) {
-            throw new GradleException("This plugin can be applied only to an Android application project")
+        boolean isAndroidLib = project.plugins.hasPlugin('com.android.library')
+
+        if (!isAndroidApp && !isAndroidLib) {
```

Comment:
fixed in https://github.com/novoda/spikes/pull/119/commits/224f6a867a40af5380f6b655f8fe07ad16c35338